src/rpc/rpcevo.cpp

```diff
@@ -30,6 +30,73 @@ extern UniValue signrawtransaction(const JSONRPCRequest& request);
 extern UniValue sendrawtransaction(const JSONRPCRequest& request);
 #endif//ENABLE_WALLET
 
+std::string GetHelpString(int nParamNum, std::string strParamName)
+{
+    std::map<std::string, std::string> mapParamHelp = {
```

Comment:
Should be `static const` as otherwise the whole map is initialized again and again.

Reply:
Good point! Fixed in 5b5eb3ae19586202597c2f79916087822f7d88cf